### PR TITLE
theme button--selected and bpk-progress

### PR DESCRIPTION
**Pull request by ejulianova**
+ [ ] For any new components I've made sure that the style can be extended by the consumer using a `className` override.
+ [ ] For any new files I've included the [Apache 2.0 License header](https://github.com/Skyscanner/backpack/blob/master/packages/bpk-tokens/formatters/license-header.js#L2-L16) at the top of the file.
+ [ ] I've updated `changelog.md` for any changes that need to be highlighted in the changelog.
+ [ ] If I've updated `npm-shrinkwrap.json` those changes are intentional.

### CODE REVIEW

File: packages/bpk-component-nudger/src/bpk-nudger.scss [hunk 1]
```diff
@@ -21,6 +21,11 @@
 .bpk-nudger {
   &__button {
     position: relative;
+    @include prop(color, secondary-color);
+
+    &[disabled] {
```
Comment (k0nserv) [inline packages/bpk-component-nudger/src/bpk-nudger.scss]: Why did you change this to not be based on class name?

Comment (ejulianova) [reply packages/bpk-component-nudger/src/bpk-nudger.scss]: The classname was being set to the icon instead of the button itself. And now we set the fill to currentColor which is the button font color.
